packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -217,7 +217,6 @@ export default function VisualEditor( { styles } ) {
 		ref,
 		useClipboardHandler(),
 		useTypewriter(),
-		useTypingObserver(),
```

Comment:
In the post editor, this hook was wrapping both the block list and the post title, now that this behavior is bundled for the block list, I kept it here around the post title.